store/smt.go

```diff
@@ -647,13 +814,13 @@ func (x *node) setChildren(leftKey, rightKey []byte) {
 	x.LeftChildKey, x.RightChildKey = leftKey, rightKey
 }
 
-// getOtherChild() returns the sibling for the child key passed
-func (x *node) getOtherChild(childKey []byte) []byte {
+// getOtherChild() returns the sibling for the child key passed and which child it is
+func (x *node) getOtherChild(childKey []byte) ([]byte, byte) {
```

Comment:
Should `other child` be an int32?

Reply:
if you mean the second argument then it can totally be, I chose `byte` as it only represent one of two possible values (left or right)